<!DOCTYPE html>
<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> 
<html> <!--<![endif]-->
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  
  <meta name="author" content="Themefisher.com">

  <title>RECOMB 2025 | ACCEPTED PAPERS</title>

  <meta name="description" content="RECOMB 2025 - Seoul, South Korea">
  <meta property="og:type" content="website">
  <meta property="og:title" content="RECOMB 2025 - Accepted papers">
  <meta property="og:image" content="images/city/yonsei-crop.jpg">
  <meta property="og:description" content="RECOMB 2025 - Seoul, South Korea">

<!-- Mobile Specific Meta
  ================================================== -->
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  <!-- Favicon -->
  <link rel="shortcut icon" type="image/x-icon" href="images/favicon.png" />
  
  <!-- CSS
  ================================================== -->
  <!-- Themefisher Icon font -->
  <link rel="stylesheet" href="plugins/themefisher-font/style.css">
  <!-- bootstrap.min css -->
  <link rel="stylesheet" href="plugins/bootstrap/css/bootstrap.min.css">
  <!-- Lightbox.min css -->
  <link rel="stylesheet" href="plugins/lightbox2/dist/css/lightbox.min.css">
  <!-- animation css -->
  <link rel="stylesheet" href="plugins/animate/animate.css">
  <!-- Slick Carousel -->
  <link rel="stylesheet" href="plugins/slick/slick.css">
  <!-- Main Stylesheet -->
  <link rel="stylesheet" href="css/style.css">  
  
  <script src="https://kit.fontawesome.com/5ce389c680.js" crossorigin="anonymous"></script>

</head>

<body id="body">

 <!-- Start Preloader
  ==================================== -->
  <div id="preloader">
    <div class='preloader'>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
    </div>
  </div> 
<!-- End Preloader
  ==================================== -->

<!-- Fixed Navigation
  ==================================== -->
  <header class="navigation fixed-top">
	<div class="container">
	  <!-- main nav -->
	  
	  <nav class="navbar navbar-expand-lg navbar-light">
		<!-- logo -->
		<a class="navbar-brand logo" href="index.html">
		  <img class="logo-default" src="images/recomb-logo/logo-name.svg" style="height:60px;" alt="RECOMB Conference Logo"/>
		  <img class="logo-white" src="images/recomb-logo/logo-name-white.svg" style="height:60px;" alt="RECOMB Conference Logo"/>
		</a>
		<!-- /logo -->
		<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navigation"
		  aria-controls="navigation" aria-expanded="false" aria-label="Toggle navigation">
		  <span class="navbar-toggler-icon"></span>
		</button>
  
		<div class="collapse navbar-collapse" id="navigation">
		  <ul class="navbar-nav mr-auto text-center">
			<li class="nav-item ">
			  <a class="nav-link" href="index.html">Home</a>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Conference</a>
			  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
				<a class="dropdown-item" href="committees.html">Committees</a>
				<a class="dropdown-item" href="keydates_and_deadlines.html">Key dates & Deadlines</a>
				<a class="dropdown-item" href="call_for_papers.html">Call for Papers</a>
				<a class="dropdown-item" href="call_for_highlights.html">Call for Highlights</a>
              <a class="dropdown-item" href="call_for_posters.html">Call for Posters</a>			  </div>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Register</a>
			  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
				<a class="dropdown-item" href="https://www.recomb2025.com/html/">RECOMB and Satellites</a>
				<a class="dropdown-item" href="travel_fellowships.html">Travel Fellowships</a>
			  </div>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Program</a>
			  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
				<a class="dropdown-item" href="program.html">Program Schedule</a>
				<a class="dropdown-item" href="keynotes.html">Keynotes</a>
				<a class="dropdown-item" href="posters.html">Poster Guidelines</a>
			  </div>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Venue</a>
			   <div class="dropdown-menu" aria-labelledby="navbarDropdown">
				<a class="dropdown-item" href="venue_main.html">Main Conference Venue</a>
				<a class="dropdown-item" href="venue_satellites.html">Satellites Venue</a>
				<a class="dropdown-item" href="hotels.html">Hotels</a>
				<a class="dropdown-item" href="airport.html">Airport Directions</a>
			<a class="dropdown-item" href="things_to_do.html">Things to do in Seoul</a>
				<a class="dropdown-item" href="transportation.html">Local Transportation</a>
			<a class="dropdown-item" href="visa.html">Visa Information</a>
			  </div>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Satellites</a>
			   <div class="dropdown-menu" aria-labelledby="navbarDropdown">
		       <a class="dropdown-item" href="satellites_schedule.html">Satellites Schedule</a>
	              <a class="dropdown-item" href="https://recomb-seq.github.io/" target="_blank">RECOMB–Seq</a>
	              <a class="dropdown-item" href="https://recomb-cg-2025.github.io/" target="_blank">RECOMB–CG</a>
	              <a class="dropdown-item" href="recombgenetics.html">RECOMB–Genetics</a>
	              <a class="dropdown-item" href="recombccb.html" target="_blank">RECOMB–CCB</a>
	              <a class="dropdown-item" href="recombrsg.html">RECOMB–CSB</a>
	              <a class="dropdown-item" href="recombmicrobiome.html">RECOMB–Microbiome</a>
	              <a class="dropdown-item" href="https://recomb-privacy-equity.github.io/" target="_blank">RECOMB–P&E</a>       </div>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Sponsors</a>
			   <div class="dropdown-menu" aria-labelledby="navbarDropdown">
				<a class="dropdown-item" href="sponsors.html">RECOMB Sponsors</a>
				<a class="dropdown-item" href="sponsorship_packages.html">Sponsorship Packages</a>
			  </div>
			</li>
			<li class="nav-item ">
			  <a href="https://www.recomb.org" target="blank" style="color:white;">About RECOMB</a>
			</li>
		  </ul>
		</div>
	  </nav>
	  <!-- /main nav -->
	</div>
  </header>
  <!-- End Fixed Navigation
	==================================== -->
<section class="single-page-header">
	<div class="container">
		<div class="row">
			<div class="col-md-12">
				<h2>Accepted Papers</h2>
			</div>
		</div>
	</div>
</section>	

<section class="team-skills section-sm" id="skills">
  <div class="container">
    <div class="col-12">
      <div class="team-skills-content center">
        <hr> <h2>RECOMB 2025 Accepted Papers</h2> <hr>
	<ul>

<li>Dynamic programming algorithms for fast and accurate cell lineage tree reconstruction from CRISPR-based lineage tracing data<br/><i>Junyan Dai and Erin Molloy</i></li><br/>
<li>Pharming: Joint Clonal Tree Reconstruction of SNV and CNA Evolution from Single-cell DNA Sequencing of Tumors<br/><i>Leah Weber, Anna Hart, Idoia Ochoa and Mohammed El-Kebir</i></li><br/>
<li>TX-Phase: Secure Phasing of Private Genomes in a Trusted Execution Environment<br/><i>Natnatee Dokmai, Kaiyuan Zhu, Cenk Sahinalp and Hyunghoon Cho</i></li><br/>
<li>mcRigor: a statistical method to enhance the rigor of metacell partitioning in single-cell data analysis<br/><i>Pan Liu and Jingyi Jessica Li</i></li><br/>
<li>Synthetic control removes spurious discoveries from double dipping in single-cell and spatial transcriptomics data analyses<br/><i>Dongyuan Song, Siqi Chen, Christy Lee, Kexin Li, Xinzhou Ge and Jingyi Jessica Li</i></li><br/>
<li>Tree reconstruction guarantees from CRISPR-Cas9 lineage tracing data using Neighbor-Joining<br/><i>Sebastian Prillo, Kevin An, Wilson Wu, Ivan Kristanto, Matthew Jones, Yun Song and Nir Yosef</i></li><br/>
<li>TarDis: Achieving Robust and Structured Disentanglement of Multiple Covariates<br/><i>Kemal Inecik, Aleyna Kara, Antony Rose, Muzlifah Haniffa and Fabian Theis</i></li><br/>
<li>Hierarchical Spatio-Temporal State-Space Modeling for fMRI Analysis<br/><i>Yuxiang Wei, Anees Abrol and Vince Calhoun</i></li><br/>
<li>devider: long-read reconstruction of many diverse haplotypes<br/><i>Jim Shaw, Christina Boucher, Yun William Yu, Noelle Noyes and Heng Li</i></li><br/>
<li>Learning multi-cellular representations of single-cell transcriptomics data enables characterization of patient-level disease states<br/><i>Tianyu Liu, Edward De Brouwer, Tony Kuo, Nathaniel Diamant, Alsu Missarova, Hanchen Wang, Minsheng Hao, Hector Corrada Bravo, Gabriele Scalia, Aviv Regev and Graham Heimberg</i></li><br/>
<li>Rag2Mol: Structure-based drug design based on Retrieval Augmented Generation<br/><i>Peidong Zhang, Xingang Peng, Rong Han, Ting Chen and Jianzhu Ma</i></li><br/>
<li>ML-MAGES: A machine learning framework for multivariate genetic association analyses with genes and effect size shrinkage<br/><i>Xiran Liu, Lorin Crawford and Sohini Ramachandran</i></li><br/>
<li>Untying rates of gene gain and loss leads to a new phylogenetic approach<br/><i>Yoav Dvir and Sagi Snir</i></li><br/>
<li>Hyper-k-mers: efficient streaming k-mers representation<br/><i>Igor Martayan, Lucas Robidou, Yoshihiro Shibuya and Antoine Limasset</i></li><br/>
<li>Decoding the Functional Interactome of Non-Model Organisms with PHILHARMONIC<br/><i>Samuel Sledzieski, Charlotte Versavel, Rohit Singh, Faith Ocitti, Kapil Devkota, Lokender Kumar, Polina Shpilker, Liza Roger, Jinkyu Yang, Nastassja Lewinski, Hollie Putnam, Bonnie Berger, Judith Klein-Seetharaman and Lenore Cowen</i></li><br/>
<li>ScatTR: Estimating the Size of Long Tandem Repeat Expansions from Short-Reads<br/><i>Rashid Al-Abri and Gamze Gürsoy</i></li><br/>
<li>TissueMosaic enables cross-sample differential analysis of spatial transcriptomics datasets through self-supervised representation learning<br/><i>Sandeep Kambhampati, Luca D'Alessio, Fedor Grab, Stephen Fleming, Fei Chen and Mehrtash Babadi</i></li><br/>
<li>Inferring cell differentiation maps from lineage tracing data<br/><i>Palash Sashittal, Richard Zhang, Benjamin Law, Alexander Strzalkowski, Henri Schmidt, Adriano Bolondi, Michelle Chan and Ben Raphael</i></li><br/>
<li>OMKar: optical map based automated karyotyping of genomes to identify constitutional abnormalities<br/><i>Siavash Raeisi Dehkordi, Zhaoyang Jia, Joey Estabrook, Jen Hauenstein, Neil Miller, Paul Dremsek, Alex Hastie, Andy Wing Chun Pang and Vineet Bafna</i></li><br/>
<li>An Exact and Fast SAT Formulation for the DCJ Distance<br/><i>Aaryan Mahesh Sarnaik, Ke Chen, Austin Diaz and Mingfu Shao</i></li><br/>
<li>GEM-Finder: dissecting GWAS variants via long-range interacting cis-regulatory elements with differentiation-specific genes<br/><i>Gyeongsik Park, Andrew J Lee and Inkyung Jung</i></li><br/>
<li>cfDecon: Accurate and Interpretable methylation-based cell type deconvolution for cell-free DNA<br/><i>Yixuan Wang, Jiayi Li, Jingqi Li, Shen Yang, Yuhan Huang, Xinyuan Liu, Yimin Fan, Irwin King, Yumei Li and Yu Li</i></li><br/>
<li>Orientation-Aware Graph Neural Networks for Protein Structure Representation Learning<br/><i>Jiahan Li, Shitong Luo, Cengyue Deng, Chaoran Cheng, Jiaqi Guan, Leonidas Guibas, Jian Peng and Jianzhu Ma</i></li><br/>
<li>Sequence-based TCR-Peptide Representations Using Cross-Epitope Contrastive Fine-tuning of Protein Language Models<br/><i>Chiho Im, Ryan Zhao, Scott Boyd and Anshul Kundaje</i></li><br/>
<li>Integer programming framework for pangenome-based genome inference<br/><i>Ghanshyam Chandra, Md Helal Hossen, Stephan Scholz, Alexander T Dilthey, Daniel Gibney and Chirag Jain</i></li><br/>
<li>DualGOFiller: A Dual-Channel Graph Neural Network with Contrastive Learning for Enhancing Function Prediction in Partially Annotated Proteins<br/><i>Shaojun Wang, Hancheng Liu, Weiqi Zhai and Shanfeng Zhu</i></li><br/>
<li>Active Learning for Protein Structure Prediction<br/><i>Zexin Xue, Michael Bailey, Abhinav Gupta, Alejandro Corrochano-Navarro, Sizhen Li, Ruijiang Li, Qiu Yu, Ziv Bar-Joseph, Sven Jager and Lorenzo Kogler-Anele</i></li><br/>
<li>A k-mer-based maximum likelihood method for estimating distances of reads to genomes enables genome-wide phylogenetic placement.<br/><i>Ali Osman Berk Şapcı and Siavash Mirarab</i></li><br/>
<li>An adversarial scheme for integrating multi-modal data on protein function<br/><i>Rami Nasser, Leah Schaffer, Trey Ideker and Roded Sharan</i></li><br/>
<li>Accurate Detection of Tandem Repeats from Error-Prone Long Reads with EquiRep<br/><i>Zhezheng Song, Tasfia Zahin, Xiang Li and Mingfu Shao</i></li><br/>
<li>Learning a CoNCISE language for small molecule binding and function<br/><i>Mert Erden, Kapil Devkota, Lia Varghese, Lenore Cowen and Rohit Singh</i></li><br/>
<li>Optimal marker genes for c-separated cell types<br/><i>Bartol Borozn, Luka Borozan, Domagoj Severdija, Domagoj Matijevic and Stefan Canzar</i></li><br/>
<li>Old dog, new tricks: Exact seeding strategy improves RNA design performances<br/><i>Théo Boury, Leonhard Sidl, Ivo L. Hofacker, Yann Ponty and Hua-Ting Yao</i></li><br/>
<li>Integration and querying of multimodal single-cell data with product-of-experts VAE<br/><i>Anastasia Litinetskaya, Maiia Shulman, Fabiola Curion, Artur Szalata, Alireza Omidi, Mohammad Lotfollahi and Fabian Theis</i></li><br/>
<li>Causal disentanglement of treatment effects in single-cell RNA Sequencing through counterfactual inference<br/><i>Shaokun An, Jae-Won Cho, Kai Cao, Jiankang Xiong, Martin Hemberg and Lin Wan</i></li><br/>
<li>ScisTree2: An Improved Method for Large-scale Inference of Cell Lineage Trees and Genotype Calling from Noisy Single Cell Data<br/><i>Haotian Zhang, Yiming Zhang, Teng Gao and Yufeng Wu</i></li><br/>
<li>Characterizing the Solution Space of Migration Histories of Metastatic Cancers with MACH2<br/><i>Mrinmoy Saha Roddur, Vikram Ramavarapu, Abi Bunkum, Ariana Huebner, Roman Mineyev, Nicholas McGranahan, Simone Zaccaria and Mohammed El-Kebir</i></li><br/>
<li>BayesRVAT: Bayesian Aggregation of Multiple Annotations Enhances Rare Variant Association Testing<br/><i>Antonio Nappi, Na Cai and Francesco Paolo Casale</i></li><br/>
<li>Dynamic mu-PBWT: Dynamic Run-length Compressed PBWT for Biobank Scale Data<br/><i>Pramesh Shakya, Ahsan Sanaullah, Degui Zhi and Shaojie Zhang</i></li><br/>
<li>The tree labeling polytope: a unified approach to ancestral reconstruction problems<br/><i>Henri Schmidt and Benjamin Raphael</i></li><br/>
<li>Joint imputation and deconvolution of gene expression across spatial transcriptomics platforms<br/><i>Hongyu Zheng, Hirak Sarkar and Benjamin Raphael</i></li><br/>
<li>STEAMBOAT: Attention-based multiscale delineation of cellular interactions in tissues<br/><i>Shaoheng Liang, Junjie Tang, Guanghan Wang and Jian Ma</i></li><br/>
<li>Learning Latent Trajectories in Developmental Time Series with Hidden-Markov Optimal Transport<br/><i>Peter Halmos, Julian Gold, Xinhao Liu and Benjamin Raphael</i></li><br/>
<li>Improved pangenomic classification accuracy with chain statistics<br/><i>Nathaniel K. Brown, Vikram Shivakumar and Ben Langmead</i></li><br/>
<li>Scalable and Interpretable Identification of Minimal Undesignable RNA Structure Motifs with Rotational Invariance<br/><i>Tianshuo Zhou, Wei Yu Tang, Apoorv Malik, David H. Mathews and Liang Huang</i></li><br/>
<li>Unified integration of spatial transcriptomics across platforms<br/><i>Ellie Haber, Ajinkya Deshpande, Jian Ma and Spencer Krieger</i></li><br/>
<li>ALPINE: an interpretable approach for decoding phenotypes from multicondition sequencing data<br/><i>Wei-Hao Lee, Lechuan Li, Ruth Dannenfelser and Vicky Yao</i></li><br/>
<li>ralphi: a deep reinforcement learning framework for haplotype assembly<br/><i>Enzo Battistella, Anant Maheshwari, Barış Ekim, Bonnie Berger and Victoria Popic</i></li><br/>
<li>A Phylogenetic Approach to Genomic Language Modeling<br/><i>Carlos Albors, Jianan Canal Li, Gonzalo Benegas, Chengzhong Ye and Yun S. Song</i></li><br/>
<li>A Partition Function Algorithm to Evaluate Inferred Subclonal Structures in Single Cell Sequencing Datasets<br/><i>Farid Rashidi Mehrabadi, Erfan Sadeqi Azer, John Bridgers, Salem Malikic, A. Funda Ergun and S. Cenk Sahinalp</i></li><br/>
<li>Prokrustean Graph: A substring index for rapid k-mer size analysis<br/><i>Adam Park and David Koslicki</i></li><br/>
<li>GeneCover: A Combinatorial Approach for Label-free Marker Gene Selection<br/><i>An Wang, Stephanie Hicks, Donald Geman and Laurent Younes</i></li><br/>
<li>Learning maximally spanning representations improves protein function annotation<br/><i>Jiaqi Luo and Yunan Luo</i></li><br/>
<li>Rewiring protein sequence and structure generative models to enhance protein stability prediction<br/><i>Ziang Li and Yunan Luo</i></li><br/>
<li>Antimicrobial drug recommendation from MALDI-TOF mass spectrometry with statistical guarantees using conformal prediction<br/><i>Nina Corvelo Benz, Lucas Miranda, Dexiong Chen, Janko Sattler and Karsten Borgwardt</i></li>

	</ul>
      </div>
      </div>
  </div> <!-- End container -->

</section> <!-- End section -->

<!-- begin Footer Area
    ========================================== -->
<footer id="footer" class="bg-one">
  <div class="top-footer">
    <p>Any questions? Email organizers at <b>recomb2025@gmail.com</b></p>
  </div>
  <div class="footer-bottom">
    <h5>Copyright 2025. All rights reserved.</h5>
    <h6>Design and Developed by Themefisher</h6>
  </div>
</footer> <!-- end footer -->
<!-- end Footer Area
    ========================================== -->

<!-- Essential Scripts
    =====================================-->
    <!-- Main jQuery -->
    <script src="plugins/jquery/jquery.min.js"></script>
    <!-- Google Map -->
    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBu5nZKbeK-WHQ70oqOWo-_4VmwOwKP9YQ"></script>
    <script  src="plugins/google-map/gmap.js"></script>

    <!-- Form Validation -->
    <script src="plugins/form-validation/jquery.form.js"></script> 
    <script src="plugins/form-validation/jquery.validate.min.js"></script>
    
    <!-- Bootstrap4 -->
    <script src="plugins/bootstrap/js/bootstrap.min.js"></script>
    <!-- Parallax -->
    <script src="plugins/parallax/jquery.parallax-1.1.3.js"></script>
    <!-- lightbox -->
    <script src="plugins/lightbox2/dist/js/lightbox.min.js"></script>
    <!-- Owl Carousel -->
    <script src="plugins/slick/slick.min.js"></script>
    <!-- filter -->
    <script src="plugins/filterizr/jquery.filterizr.min.js"></script>
    <!-- Smooth Scroll js -->
    <script src="plugins/smooth-scroll/smooth-scroll.min.js"></script>
    
    <!-- Custom js -->
    <script src="js/script.js"></script>

  </body>
  </html>
